js/modules/handleBurgerButtons.js

```diff
@@ -0,0 +1,30 @@
+const menuOverlay = document.body.querySelector(".menu_overlay");
+const burgerBtn = document.body.querySelector(".menu_burger");
+const menuCloseBtn = menuOverlay.querySelector(".menu_close_btn");
+
+burgerBtn.addEventListener("click", openMenu);
+
+function openMenu(){
+	menuOverlay.classList.add("active");
+	document.body.classList.add("stopPageScroll");
+	document.addEventListener("click", closeMenuByOverlay);
```

Comment:
why do you need `document ` for it instead of `menuOverlay` ?

Reply:
Done.

Reply:
place addEventListeners otside the function both on open and close, didn't you understand the principle?
I've pointed out such places already several times.

Reply:
Done.